### ISSUE TITLE
SuperCardPro image format 1.6

### ISSUE THREAD

**Issue by claunia** [simonowen/samdisk#2]
Hi Simon,

Maybe you're interesting in supporting new SCP 1.6 feature (footer with some metadata):
http://www.cbmstuff.com/downloads/scp/scp_image_specs.txt

I made a PR for https://github.com/keirf/Disk-Utilities/pull/56/commits/a4cf19c1db4dafdc1901d50e6c3bc252a983d36b and also added support in https://github.com/claunia/DiscImageChef/commit/c50b49ddd1ee791c8172a6b03eb271794c09f6fd

If you are too busy I can try to make you a PR if you want.

Regards!

**Comment by simonowen** [comment]
I'm certainly interested in supporting it, but haven't read the updated spec details yet.  However, looking at your PR for Disk-Utilities it shouldn't be too difficult to add in.

I'll leave this issue open for now and update it before I start looking at it.  Hopefully that avoids any work duplication if someone beats me to it.  :)

Thanks!

**Comment by simonowen** [comment]
I've added this to .scp reading, but it'd be useful to check it with a real image before I commit it.  Do you have a sample image that makes use of the extra fields?  Thanks again!

**Comment by simonowen** [comment]
@claunia Are the timestamps UTC, or local time for whoever created the image?

**Comment by claunia** [comment]
```
BYTES 0x18-0x1F contains the little-endian signed 64-bit count of seconds since
1st January 1970 00:00:00 in UTC
```

**Comment by simonowen** [comment]
I somehow completely missed your detailed description in the official spec.  I've implemented it in 7907cd24e1fc08b531de5ff11407d731b9204d9d.

Thanks for your help!

**Comment by claunia** [comment]
Great!
